src/row_objects.ts

```diff
@@ -0,0 +1,66 @@
+import { Rows, Empty, ColumnName } from "./rows.ts";
+
+export class RowObjects<T extends any = Record<string, any>> {
```

Comment:
I don't think this should be generic, since the type never changes 😅

Reply:
Actually, by having this generic the user can use their own types, e.g. `asObjects<CustomObject>`, I could maybe change the signature though, what do you think? 🤔

Reply:
Ohhh, I see 👀 Yeah, you're right, I didn't think of that, let's keep the type parameter 😅